run.go

```diff
@@ -156,13 +157,16 @@ func (r *runner) concurrent(rate int) (failed bool) {
 	return
 }
 
-func runWithOptions(suiteName string, runner runner, opt Options) int {
-	var output io.Writer = os.Stdout
-	if nil != opt.Output {
-		output = opt.Output
+func configureFormatter(opt Options, suiteName string, output io.WriteCloser) (Formatter, error) {
+	multiFmt, err := configureMultiFormatter(opt, output)
+	if err != nil {
+		return nil, err
 	}
 
-	multiFmt := ifmt.MultiFormatter{}
+	return multiFmt.FormatterFunc(suiteName, output), nil
+}
+
+func configureMultiFormatter(opt Options, output io.WriteCloser) (multiFmt ifmt.MultiFormatter, err error) {
```

Comment:
`nit`: Consider dropping use of named return values (cost/benefit)?

Reply:
agree